lib/index.js

```diff
@@ -378,7 +397,10 @@ exports.processDir = function(dir, options) {
     options.baseDir = dir;
   }
   const pattern = options.files || defaults.files;
-  return promisify(glob)(path.join(dir, pattern))
+
+  const getFilesPath = async pathPattern => await glob(pathPattern)
+
+  return asyncPromisify(getFilesPath)(path.join(dir, pattern))
```

Comment:
also we can refactor this function to use async await and avoid [`asyncPromisify`](https://github.com/claudetech/node-static-i18n/pull/56/files#diff-92bbac9a308cd5fcf9db165841f2d90ce981baddcb2b1e26cfff170929af3bd1R63)